pvanalytics/tests/quality/test_data_shifts.py

```diff
@@ -0,0 +1,82 @@
+"""Tests for data shift quality control functions."""
+import pandas as pd
+import ruptures as rpt
+import pytest
+from pvanalytics.quality import data_shifts as dt
+
+
+@pytest.fixture
+def generate_daily_time_series():
+    # Pull down the saved PVLib dataframe and process it
+    df = pd.read_csv("https://datahub.duramat.org/dataset/7b72ae24-c0c2-4339"
```

Comment:
I wonder if we should bundle this file with the package instead of downloading it?  It's 142 KiB, so not huge.  I imagine future tests might want their own datasets, so maybe we should add a `pvanalytics/data` folder like pvlib has.

Reply:
+1 to including the file and to setting up `pvanalytics/data`.